### PR TITLE
Fix horizontal scrollbars on Safari when code blocks are very long

### PR DESCRIPTION
**Pull request by dobromir-hristov**
Bug/issue #, if applicable: 110741983

## Summary

Fixes a bug on Safari, where if the last line in a codeblock is very long, the `after` accessibility pseudo element helper gets pushed to the side, causing the browser to render hirozontal scrollbars.

## Dependencies

NA

## Testing

Load a page with a codeblock that has very long last line

Steps:
1. Assert in Safari there are no horizontal scrollbars

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [ ] ~Added tests~ - css only
- [ ] Ran `npm test`, and it succeeded
- [ ] Updated documentation if necessary

### CODE REVIEW

**Review by mportiz08 — APPROVED** [review]
Awesome, thanks!

@hqhhuang can you also test this with the edge case that you found?

**Review by hqhhuang — APPROVED** [review]
Tested and looks great with the minimized edge case as well.

Did a quick search for pseudo elements in other parts of the code base. Should we add this fix here as well?
https://github.com/apple/swift-docc-render/blob/715eb13b2fed2da840e7a8eda41ffec8248dd3c4/src/components/ContentNode/StrikeThrough.vue#L34 
I couldn't find an example of where we use this component to test though.

**Comment by mportiz08** [comment]
> Tested and looks great with the minimized edge case as well.
> 
> Did a quick search for pseudo elements in other parts of the code base. Should we add this fix here as well?
> 
> https://github.com/apple/swift-docc-render/blob/715eb13b2fed2da840e7a8eda41ffec8248dd3c4/src/components/ContentNode/StrikeThrough.vue#L34
> 
> 
> I couldn't find an example of where we use this component to test though.

I don't believe so. The strikethrough is used for inline text that looks like ~~this~~ and we wouldn't want to use a block display there.

File: src/components/CodeBlock.vue [hunk 1]
```diff
@@ -36,6 +36,8 @@ code {
     content: attr(data-after-code);
   }
   &::before, &::after {
+    // ensure the pseudo elements dont fly off in space
+    display: block;
```
[the remark this answers is not in the document]
Comment (hqhhuang) [reply src/components/CodeBlock.vue:40]: Maybe a Safari bug? Running the version that has regression, In the inspector, display is set to block by default.
image

**Comment by mportiz08** [comment]
@swift-ci test
